src/main/resources/lib/metadata/index.js

```diff
@@ -138,4 +131,35 @@ function getReusableData(contentPath=undefined) {
     }
 
     return { site, content, siteConfig };
-}
\ No newline at end of file
+}
+
+module.exports = {
+    getReusableData,
+    getMetaData,
+    getTitle,
+    getFixedHtmlAttrsAsString
+}
+
+// Functions below module.exports are only used internally
+
+function resolveMetadata(params, selfClosingTags=false) {
+    const metadataWithNonClosingTags = libs.thymeleaf.render(resolve("metadata.html"), params);
+
+    if ( !selfClosingTags ) {
+        return metadataWithNonClosingTags;
+    }
+
+    const metaTagsRegexp = new RegExp("<meta (.*?)>", "g");
```

Comment:
parsing html with regexp is error prone. 
`<meta name="description" content=">">`

Reply:
Good catch!

I'd like to avoid installing a npm package just to do this parsing... What do you suggest?

I was thinking about using a polyfill for it, such as [this one](https://gist.github.com/eligrey/1129031) or [this other one](https://johnresig.com/files/htmlparser.js).